aiogram/bot/bot.py

```diff
@@ -310,6 +310,8 @@ async def send_message(self,
         payload = generate_payload(**locals())
         if self.parse_mode and entities is None:
             payload.setdefault('parse_mode', self.parse_mode)
+        if self.disable_web_page_preview and entities is None:
```

Comment:
I don't understand why you check whether entities are not specified. It's needed only for `parse_mode`.

```suggestion
        if self.disable_web_page_preview:
```

Reply:
done! fixed